src/CitMovie.Api/Controller/CompletedController.cs

```diff
@@ -75,47 +76,8 @@ public async Task<IActionResult> GetUserCompleted(int userId, int page = 0, int
         return Ok(result);
     }
 
-    [HttpPatch("{id}")]
-    public async Task<IActionResult> UpdateCompleted(int userId, int id, [FromBody] string? note)
-    {
-        if (userId != GetUserId())
-            return Forbid();
-
-        if (note == null)
-            return BadRequest("Note content is required.");
-
-        var completed = await _completedManager.GetCompletedAsync(id);
-        if (completed == null)
-            return NotFound();
-
-        if (completed.UserId != userId)
-            return Forbid();
-
-        var updateCompletedDto = new UpdateCompletedDto { Note = note };
-        var updatedCompleted = await _completedManager.UpdateCompletedAsync(id, updateCompletedDto);
-        return updatedCompleted == null ? NotFound() : Ok(updatedCompleted);
-    }
-
-    [HttpDelete("{id}")]
-    public async Task<IActionResult> DeleteCompleted(int userId, int id)
-    {
-        if (userId != GetUserId())
-            return Forbid();
-
-        var completed = await _completedManager.GetCompletedAsync(id);
-        if (completed == null)
-            return NotFound();
-
-        if (completed.UserId != userId)
-            return Forbid();
-
-        var deleted = await _completedManager.DeleteCompletedAsync(id);
-        return deleted ? NoContent() : NotFound();
-    }
-
```

Comment:
Not really sure why this has been removed? should it not be possible to delete a movie if you added it by mistake or update it?

Reply:
Will add it back.